test/runtests.jl

```diff
@@ -2,36 +2,38 @@ using Test
 using PSDMatrices
 using LinearAlgebra
 
-M1 = [1 1; 2 20]
-M2 = [1.0 1.0; 2.0 20.0]
-M3 = big.([1.0 1.0; 2.0 20.0])
+Mbase = [1 1; 2 20]
+eltypes = (Int64, Float64, BigFloat)
 
-@testset "$M" for M in (M1, M2, M3)
-    S = PSDMatrix(M)
+@testset "PSDMatrices.jl" begin
+    @testset "eltype=$t" for t in eltypes
+        M = t.(Mbase)
+        S = PSDMatrix(M)
 
-    @test eltype(S) == eltype(M)
+        @test eltype(S) == eltype(M) == t
```

Comment:
```suggestion
        @test eltype(S) == t
```
I think the `eltype(M) == t` bit is a bit redundant

Reply:
But I might be incorrect.

Reply:
Yes true, that should hold here in any case.